var/spack/repos/builtin/packages/eccodes/package.py

```diff
@@ -0,0 +1,87 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+#
+from spack import *
+
+
+class Eccodes(CMakePackage):
+    """ecCodes is a package developed by ECMWF which provides an application
+    programming interface and a set of tools for decoding and encoding
+    messages in the following formats:
+
+      WMO FM-92 GRIB edition 1 and edition 2
+      WMO FM-94 BUFR edition 3 and edition 4 
+      WMO GTS abbreviated header (only decoding).
+    """
```

Comment:
If you run `spack info eccodes` or build the documentation you'll find that newline characters aren't preserved, so this will be formatted strangely. You might want to add more punctuation.